### DISCUSSION TITLE
create alert

### DISCUSSION THREAD

**Discussion by YALouis** [nextgenhealthcare/connect#5778] [Q&A] [answered]
What can I do with alert?I want to indicate when the channel is not processing new data.Whether the alert can achieve such a function?

**Comment by pacmano1** [comment]
Do you mean "alert"?

**Reply by YALouis** [reply]
yes,sorry for that..

**Reply by pacmano1** [reply]
The Advanced Alerting plugin (which is not free) handles this.   If you do not have that extenstion, create a channel that calls mirth's own API to get message counts and alert as per your needs.

**Reply by YALouis** [reply]
Is there some documents on what apis are available

**Reply by pacmano1** [reply]
It's a menu item on the bottom left of the mirth admin tool, the user and client API menu items.

**Reply by YALouis** [reply]
Wow,I found lots of very useful interfaces, thank you, but I tried to test them with postman and it said 401,unauthorized, what I should set up. I can get response using a web page.

**Reply by pacmano1** [reply, marked as answer]
Post no screenshots on what you tried, get no answers.

The calls use basic authorization.   Recent versions of Mirth required the "X-Requested-With" header.  Pleaase just transport the curl statement the API tester shows which is at https://your-mirth-instance-ip:8443/api/, it is that easy, remember to login on the upper right of that page.    Please note the required fields are not correct on some calls, see other posts on that.

Please fix your spelling on this discussion.
